spec/spec_helper_acceptance.rb

```diff
@@ -24,6 +24,16 @@
         # CentOS has epel-release package in Extras, enabled by default
         shell('yum -y install epel-release')
       end
+      puts "platform is #{host['platform']}"
+      if host['platform'] =~ /windows/
+        require 'winrm'
+        include Serverspec::Helper::Windows
+        include Serverspec::Helper::WinRM
+
+        endpoint = "http://127.0.0.1:5985/wsman"
+        c.winrm = ::WinRM::WinRMWebService.new(endpoint, :ssl, :user => 'vagrant', :pass => 'vagrant', :basic_auth_only => true)
```

Comment:
After commenting out Serverspec includes, get error below.  Why use `winrm`?  Beaker is able to access the Windows VM without `winrm`.

```
An error occurred in a `before(:suite)` hook.
Failure/Error: c.winrm = ::WinRM::WinRMWebService.new(endpoint, :ssl, :user => 'vagrant', :pass => 'vagrant', :basic_auth_only => true)
NameError:
  uninitialized constant WinRM::WinRMWebService
  Did you mean?  WinRM::WinRMWMIError

# ./spec/spec_helper_acceptance.rb:34:in `block (3 levels) in <top (required)>'
# ./spec/spec_helper_acceptance.rb:22:in `each'
# ./spec/spec_helper_acceptance.rb:22:in `block (2 levels) in <top (required)>'
```

FWIW I got beaker further with this change:

```
         endpoint = "http://127.0.0.1:5985/wsman"
-        c.winrm = ::WinRM::WinRMWebService.new(endpoint, :ssl, :user => 'vagrant', :pass => 'vagrant', :basic_auth_only => true)
-        c.winrm.set_timeout 300
+        c.winrm = ::WinRM::Connection.new(endpoint: endpoint, transport: :ssl, user: 'vagrant', password: 'vagrant', basic_auth_only: true)
+        #c.winrm.set_timeout 300
```

Reply:
We don't need to use WinRM. Just my first take on getting it to work.